server.py

```diff
@@ -22,20 +22,29 @@
 
 app = Flask(__name__)
 
-class DoorOpener(object):
+class DoorManager(object):
     def __init__(self):
         self.open_door_ts = 0.0
```

Comment:
Style Request: Underscores on the variable names `_open_door_ts`. If you make a separate PR with the underscores, I'll just !accept it immediately so that this one gets less cluttered.